source/dbo/Stored Procedures/Power BI Reporting/uspGenerateNtbsCaseRecord.sql

```diff
@@ -15,6 +29,25 @@ BEGIN TRY
 		WHERE rr.SourceSystem = 'NTBS'
 		GROUP BY rr.NotificationId;
 
+	INSERT INTO @TempManualTestResult
+	SELECT DISTINCT rr.NotificationId, ManualTestTypeId,
```

Comment:
You could use a where clause to exclude Chest x-ray (and the upcoming Chest CT) test types here, as you won't be trying to summarise them for this anyway?

Reply:
Yep added where clause